### ISSUE TITLE
Error searching for linters on Windows

### ISSUE THREAD

**Issue by giggio** [fnando/vscode-linter#11]
When searching for tools on Windows the path is incorrect. It seems to be using Linux's path separator `:`, instead of Windows' `;`.

Sample output:

```
[5/19/2022, 4:43:12 PM] The yamllint binary couldn't be found within $PATH: [
  "C",
  "\\Program Files\\PowerShell\\7;C",
  "\\Windows\\System32;C",
  "\\Windows;C",
...
```

**Comment by HackproTm** [comment]
The PR [#10](https://github.com/fnando/vscode-linter/pull/10) solves this issue.

**Comment by fnando** [comment]
Released as v0.0.12
